test/plugin/test_in_forward.rb

```diff
@@ -465,6 +469,68 @@ def test_skip_invalid_event(data)
     end
   end
 
+  class CompressedPackedForward < self
+    extend Fluent::Test::StartupShutdown
+
+    def test_set_compress_to_option
+      d = create_driver
+
+      time = Fluent::EventTime.parse("2011-01-02 13:14:15 UTC").to_i
```

Comment:
`event_time` helper function is available (defined in `lib/fluent/test/helpers.rb`).
E.g., `event_time("2011-01-02 13:14:15 UTC")`.